### DISCUSSION TITLE
How is this Board programmed? USB or JTAG?

### DISCUSSION THREAD

**Discussion by ErichHeinemann** [marcel-licence/esp32_midi_sampler#5] [Q&A]
Hi Marcel, 
perhaps a stupid question, but I didn´t find an answer with google.
Do I need a JTAG-Programmer to program the ESP32?
Or does it work via the USB-UART?

Best Regards
Erich

**Comment by marcel-licence** [comment]
Hi Erich,
this board will be recognized as a serial device (USB-UART -> there is a label on the connector). It can be programmed like other ESP's via arduino.
Select Board: "ESP32 Dev Module".
Set PSRAM: "Enabled"
Then you can upload your sketchfile.
Best regards,
Marcel
